\title{seqr : a web-based analysis and collaboration tool for rare disease genomics}

\section{abstract}
Exome and genome sequencing have become the tools of choice for rare disease diagnosis, leading to large amounts of data available for analyses. To identify causal variants in these datasets, powerful filtering and decision support tools that can be efficiently used by clinicians and researchers are required. To address this need, we developed seqr - an open source, web-based tool for family-based monogenic disease analysis that allows researchers to work collaboratively to search and annotate genomic callsets. To date, seqr is being used in several research pipelines and one clinical diagnostic lab. In our own experience through the Broad Institute Center for Mendelian Genomics, seqr has enabled analyses of over 10,000 families, supporting the diagnosis of more than 3,800 individuals with rare disease and discovery of over 300 novel disease genes. Here we describe a framework for genomic analysis in rare disease that leverages seqr’s capabilities for variant filtration, annotation, and causal variant identification, as well as support for research collaboration and data sharing. The seqr platform is available as open source software, allowing low-cost participation in rare disease research, and a community effort to support diagnosis and gene discovery in rare disease.

\section{1. Introduction}
Approximately 1 in 20 people worldwide are affected by a rare genetic condition, but approximately 65% of cases go undiagnosed due to limitations in diagnostic technology used, a lack of understanding of human genomic variation, and insufficient delineation of the mechanisms underlying disease (Chong et al., 2015; Boycott et al., 2017). Many of these unsolved cases move into the research realm, where exome and genome sequencing have been shown to increase the diagnostic yield by identifying complex variants and novel causes of disease (Clark et al., 2018; Palmer et al., 2021). With the large-scale uptake of exome and genome sequencing by research programs, including the USA’s Centers for Mendelian Genomics (CMG) (Chong et al., 2015) and Undiagnosed Disease Network (UDN) (Gahl et al., 2012), Canada’s Care4Rare program (care4rare.ca), and the UK’s Deciphering of Developmental Disorders (DDD) project (Wright et al., 2015), and Genomics England 100,000 Genomes study (Caulfield et al., 2017) among many others, vast and rapidly growing amounts of data are now available for analysis. To identify disease-causing variants in these large datasets, powerful filtering and decision support tools that can be easily accessed and used by researchers are needed.